mm/kasan/CMakeLists.txt

```diff
@@ -22,12 +22,9 @@
 set(SRCS hook.c)
 
 if(CONFIG_MM_KASAN)
+  list(APPEND FLAGS -fno-lto)
```

Comment:
May break the GHS toolchain, Reference:
https://github.com/apache/nuttx/blob/master/arch/arm/src/cmake/ghs.cmake#L52
https://github.com/apache/nuttx/blob/master/libs/libc/assert/CMakeLists.txt#L26-L29

Reply:
But this citest configuration does not turn on kasan?
![Uploading image.png…]()